src/seedsigner/models/seed.py

```diff
@@ -239,3 +239,61 @@ def seedqr_supported(self) -> bool:
     @property
     def bip85_supported(self) -> bool:
         return False
+
+
+
+class ShamirSeed(Seed):
+    def __init__(self,
+                 mnemonics: List[str] = None,
+                 passphrase: str = "",
+                 wordlist_language_code: str = SettingsConstants.WORDLIST_LANGUAGE__ENGLISH) -> None:
```

Comment:
It feels less and less likely that we'll ever support the non-English bip39 wordlists. And so it feels even less likely that we'd support the slip39 equivalents.

I would just drop the wordlist flexibility entirely. And the rest of our bip39 code will probably do the same in a future release.